### PR TITLE
Update CHS translation

### PR DESCRIPTION
**Pull request by maxsky**

### CODE REVIEW

File: Translations/Web/lang_zh-cn.php [hunk 1]
```diff
@@ -62,9 +62,14 @@
...
 // Privacy Policy
 $content_privacypolicy_title = '隐私政策';
+$content_privacypolicy_content = 'Keka 不会从您的计算机发送或存储任何类型的数据。';
+$content_privacypolicy_MAS = 'MAS 版本不存在任何网络功能。';
+$content_privacypolicy_WEB = 'WEB 版本将 Sparkle 用于更新过程（可禁用），所以该组件可以连接到网络。它仅用于检查新版本并下载。该组件从未存储或共享任何类型的信息，甚至是在 Keka 1.0.0 之前的某些版本中启用但从未使用过的<a href="https://sparkle-project.org/documentation/system-profiling/">匿名系统配置文件数据</a>。';
 
 // Terms of use
 $content_termsofuse_title = '使用条款';
+$content_termsofuse_inapp_title = '应用内购';
+$content_termsofuse_inapp = 'Keka 的 App Store 版本提供应用购买，作为捐赠和帮助项目开发的一种方式, 与 <a href="https://www.keka.io/#lovekeka">网站中的 PayPal/Sponsors</a> 相似。这些购买项是可选的且不授予任何其它功能。';
```
Comment (aonez) [inline Translations/Web/lang_zh-cn.php:72]: I was not thinking of translating those, that's why I've omitted them from the translation files. But I'll accept those anyway @maxsky ;)

File: Translations/zh-Hans.lproj/main.strings [hunk 2]
```diff
@@ -90,7 +90,7 @@
 "maF-B5-Hb6.title" = "关于 Keka";
 
 /* Class = "NSMenuItem"; title = "Revert to Saved"; ObjectID = "nlc-UW-DRW"; */
-"nlc-UW-DRW.title" = "复原到已存储的版本";
+"nlc-UW-DRW.title" = "还原至已保存";
```
Comment (aonez) [inline Translations/zh-Hans.lproj/main.strings:93]: Is this and the next below used in Big Sur?

File: Translations/zh-Hans.lproj/main.strings [hunk 3]
```diff
@@ -117,7 +117,7 @@
 "tjx-uw-4ov.title" = "页面设置…";
 
 /* Class = "NSMenuItem"; title = "Copy"; ObjectID = "v5b-pg-pYC"; */
-"v5b-pg-pYC.title" = "拷贝";
+"v5b-pg-pYC.title" = "复制";
```
Comment (aonez) [inline Translations/zh-Hans.lproj/main.strings:120]: I'm seeing the former used in Catalina, is this one preferred in Big Sur?

**Review by aonez — CHANGES_REQUESTED** [review]
Just a question!

File: Translations/Web/lang_zh-cn.php (hunk 1 above)
Comment (maxsky) [reply Translations/Web/lang_zh-cn.php:72]: ;) I also thought about it, but finally decided to translate.

File: Translations/zh-Hans.lproj/main.strings (hunk 2 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings:93]: I not found it in Big Sur...just fixed of old translation.

File: Translations/zh-Hans.lproj/main.strings (hunk 3 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings:120]: The meaning of the two expressions is the same, and the new translation is more reasonable.

File: Translations/zh-Hans.lproj/main.strings (hunk 2 above)
Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:93]: True, It is no present, neither in Catalina

File: Translations/zh-Hans.lproj/main.strings (hunk 3 above)
Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings:120]: And in Catalina, apple used "复制" as "Copy" translation.

Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:120]: Where? I've checked in Catalina and Big Sur and found the previous translation.

Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:120]: At least on Finder in Big Sur and Catalina:

Screenshot 2021-01-25 at 10 24 49
Screenshot 2021-01-25 at 10 25 54

Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:120]: I see what happened. You're talking about the one in the context menu:

Screenshot 2021-01-25 at 10 30 14

But the Keka one is the "Copy" from the "Edit" menu. The finder one is a "Duplicate" and has a different function.

Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings:120]: OIC! I just check File menu, sorry I forgot it in Edit menu. Yes it should use "拷贝".

Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings:120]: let me update, i found a small problem.

Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:120]: No problem! I just was curious if it changed in Big Sur 🤗 Better stick with the old one then.

Comment (aonez) [reply Translations/zh-Hans.lproj/main.strings:120]: @maxsky there was no need to close the pull, just updating your repo updated this pull 😅

Comment (maxsky) [reply Translations/zh-Hans.lproj/main.strings]: 😂lol, awkward

**Comment by aonez** [comment]
Thanks!!! Hopefully I'm pushing this tomorrow or on Thursday 😉

**Review by aonez — APPROVED** [review]

File: Translations/zh-Hans.lproj/Localizable.strings [hunk 4]
```diff
@@ -125,7 +125,7 @@
 "unknown" = "未知";
 "Unknown error" = "未知错误";
 "Error code %d using \"%@\"\n%@" = "错误码 %d 使用“%@”\n%@";
-"Keka was unable to create the extraction package" = "Keka was unable to create the extraction package";
+"Keka was unable to create the extraction package" = "Keka 无法创建压缩包";
```
Comment (aonez) [inline Translations/zh-Hans.lproj/Localizable.strings:128]: This is about the "extraction" package, where the extraction will go. Is that one right?

Comment (maxsky) [reply Translations/zh-Hans.lproj/Localizable.strings:128]: Is there has a detailed context description?

Comment (aonez) [reply Translations/zh-Hans.lproj/Localizable.strings:128]: While extracting a compressed file a ".kextraction" package is created, and there the extraction goes while unfinished. Once finished, it's moved to the proper extracted files. Imagine "the extraction package" as a temporary place to store the extraction files while they are being extracted.

Screenshot 2021-01-25 at 10 54 35

Comment (maxsky) [reply Translations/zh-Hans.lproj/Localizable.strings:128]: OK, I am understanding it. Let me think about how to translate it of "extraction package".

File: Translations/zh-Hans.lproj/Localizable.strings [hunk 5]
```diff
@@ -214,21 +214,21 @@ Apple needs to have MAS translated in this language so this strings can be used
...
-"Tip a dinner" = "Tip a dinner";
-"Tip Keka a dinner so it keeps rolling" = "Tip Keka a dinner so it keeps rolling";
-"Tip a bunch of food" = "Tip a bunch of food";
-"Tip Keka a new bunch of food to keep rolling" = "Tip Keka a new bunch of food to keep rolling";
+"Monthly sip of coffee" = "每月整点☕";
+"Gift Keka a sip of coffee each month" = "每月请 Keka 整点☕";
+"Monthly coffee" = "月供☕";
+"Gift Keka a coffee each month" = "每月请 Keka 整一杯☕";
+"Monthly pair of coffees" = "每月两杯☕";
+"Gift Keka a couple coffees each month" = "每月请 Keka 整几杯☕";
```
Comment (aonez) [inline Translations/zh-Hans.lproj/Localizable.strings:222]: Sorry @maxsky, just realized this is using an emoji. This can't be used in MAS 😞

**Pull request merged**